lib/sequelize.d.ts

```diff
@@ -249,6 +249,13 @@ export interface Options {
    * Defaults to false
    */
   typeValidation?: boolean;
+
+  /**
+   * String based operator alias, default value is true which will enable all operators alias. Pass object to limit set of aliased operators or false to disable completely.
+   *
+   * Defaults to true
+   */
+  operatorsAliases?: boolean | object;
```

Comment:
Please use a more precise type than `object`, e.g. an index signature or an interface.

Reply:
It seems that creating an interface here would require porting the OperatorAliases into an Typescipt interface, then referencing that interface in our `sequelize.d.ts` file. Does that seem like a reasonable approach? Here are the operator aliases from the Sequelize source code: https://github.com/sequelize/sequelize/blob/master/lib/operators.js 

I was hoping this would have already been done in the other Sequelize type definitions, but I can't seem to find it here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/sequelize/v3/index.d.ts I'm happy to implement this port when I have some time.

Reply:
They are all here: https://github.com/types/sequelize/blob/ff2141546a12568a4a3909a561c1ff7971112d20/lib/model.d.ts#L100

Should be possible to use a mapped type.

DefinitelyTyped has nothing to do with this repo.